apollo-ios-codegen/Sources/ApolloCodegenLib/CodegenConfiguration/ApolloCodegenConfiguration.swift

```diff
@@ -283,7 +283,12 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
       case embeddedInTarget(name: String, accessModifier: AccessModifier = .internal)
       /// Generates a `Package.swift` file that is suitable for linking the generated schema types
       /// files to your project using Swift Package Manager.
+      @available(*, deprecated, message: "Use .swiftPackage(dependencyType:) case instead.")
       case swiftPackageManager
+      /// Generates a `Package.swift` file that is suitable for linking then generated schema types
+      /// files to your project using Swift Package Manager. Uses the `dependencyType` associated
+      /// value to determine how to setup the dependency on `apollo-ios`.
+      case swiftPackage(dependencyType: ApolloDependencyType = .default)
```

Comment:
It still isn't clear to me that this is the dependency on `apollo-ios` without reading the documentation. I think something like `apolloSDKVersion` makes it clear that we are not talking about the "dependency type" of your schema module.

Reply:
I'm catching up on the changes now and I came here to make the same comment. @AnthonyMDev's naming suggestion certainly makes it clearer to the user which version they're configuring.